2-mandatory/2-water-bottle.js

```diff
@@ -15,22 +15,24 @@ You have to implement the missing features according to the specification.
 */
 
 // Here is your starting point:
-let bottle = {
+const bottle = {
   volume: 0,
-  fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+  fillUp() {
+    this.volume = 100;
   },
-  pour: function () {
-    // calling this function should increase your bottle volume by 10 units;
+  pour() {
+    if (this.isFull()) return;
+    this.volume = this.volume > 90 ? 100 : this.volume + 10;
   },
-  drink: function () {
-    // calling this function should decrease your bottle volume by 10 units;
+  drink() {
+    if (this.isEmpty()) return;
+    this.volume = this.volume < 10 ? 0 : this.volume - 10;
   },
```

Comment:
You are still using magic numbers here, which is not recommended. Can you think of a way of resolving the problem that doesn't use or minimizes the use of magic numbers?

Reply:
Should be fixed now, thanks!